sgt-back-end/server.ts

```diff
@@ -0,0 +1,141 @@
+import pg from 'pg';
+import express from 'express';
+import { ClientError } from './client-error.js';
+import { errorMiddleware } from './error-middleware.js';
+
+const app = express();
+
+// only create ONE pool for your whole server
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    // Allow non-SSL traffic to localhost
+    rejectUnauthorized: false,
+  },
+});
+
+// type Grade = {
+//   course: string;
+//   createdAt: string;
+//   gradeId: number;
+//   name: string;
+//   score: number;
+// };
+
+app.use(express.json());
+
+app.get('/api/grades', async (req, res, next) => {
+  try {
+    const sql = `
+      select *
+        from "grades";
+    `;
+
+    const grades = await db.query(sql);
```

Comment:
If you'd like, you can use `db.query<Grade>(sql)` to have TS understand that the result will be a `Grade[]`. That can provide some extra type checking, though not really all that much since we don't usually do much with the result!